src/routes/api/index.js

```diff
@@ -15,5 +15,8 @@ router.use('/api/v1/auth', userRoutes);
 // set up auth routes v1
 router.use('/api/v1/auth', userRoutes);
 
+// Assign user roles
+router.use('/api/v1/', userRoutes);
```

Comment:
```suggestion
router.use('/api/v1/auth', userRoutes);
```

Reply:
@blaise82 again what am working on is not auth so am not supposed to be using that prefix

Reply:
@blaise82 thanks I found out what you meant.

Reply:
great
@AbdoulNuru  thanks for understanding